tests/ui/autodiff/autodiff_illegal.stderr

```diff
@@ -19,32 +19,24 @@ error: expected 1 activities, but found 2
    |
 LL | #[autodiff(df3, Reverse, Duplicated, Const)]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-   |
-   = note: this error originates in the attribute macro `autodiff` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
I am not 100% sure why this message disappeared, probably I use a different span somewhere?
I don't think it gives valuable information here, so I'm fine with less noise, any objections?